Dockerfile

```diff
@@ -18,5 +18,6 @@ RUN pip install lxml scrapy scrapyjs
 # Define environment variable
 #ENV NAME world
 
+WORKDIR /app/accounts
 #Run app.py when the container launches
-CMD ["python", "app.py"]
+CMD ["scrapy", "crawl", "spider"]
```

Comment:
I changed this because I didn't know if you were using Docker, but I did it my way.

Now doing 
```
docker build -t ingcrawler .
docker run ingcrawler
```

will run the crawler

Reply:
Feel free to revoke those changes